handbook/product/index.md

```diff
@@ -12,9 +12,9 @@ The product team consists of the product managers, product designers, and techni
   - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) - ([Web](../engineering/web/index.md))
   - [Malo Marrec](../../../company/team/index.md#malo-marrec-he-him) - ([Campaigns](../engineering/campaigns/index.md))
 - [Product design](design/index.md)
-  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
-  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
-  - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+  - [Rob Rhyne](../../../company/team/index.md#rob-rhyne) - ([Campaigns](../engineering/campaigns/index.md)) and ([Search](../engineering/search/index.md))
+  - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) - ([Web](../engineering/web/index.md)) and ([Code Insights](../engineering/code-insights/index.md))
```

Comment:
@AlicjaSuska I saw you added yourself to the code intel team (yay! :tada:) should team also be added here?

Reply:
Just added, good point, thank you @macraig 🎉